cg/constants/demultiplexing.py

```diff
@@ -71,12 +71,18 @@ class RunParametersXMLNodes(StrEnum):
 class SampleSheetBcl2FastqSections:
     """Class with all necessary constants for building a NovaSeqX sample sheet."""
 
-    class Settings(Enum):
+    class Settings(StrEnum):
         HEADER: str = "[Settings]"
-        BARCODE_MISMATCH_INDEX1: list[str] = ["BarcodeMismatchesIndex1", "0"]
-        BARCODE_MISMATCH_INDEX2: list[str] = ["BarcodeMismatchesIndex2", "0"]
 
-    class Data(Enum):
+        @classmethod
+        def barcode_mismatch_index_1(cls) -> list[str]:
+            return ["BarcodeMismatchesIndex1", "0"]
+
+        @classmethod
+        def barcode_mismatch_index_2(cls) -> list[str]:
+            return ["BarcodeMismatchesIndex2", "0"]
```

Comment:
Slightly weird to have as class methods when they are independent of any class values, but probably worth it for consistent pattern.

Reply:
Agreed - but that is how I found them. Can be changed later if we want to.